app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,30 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
+    @user = User.find(params[:user_id])
   end
```

Comment:
- 
![Selection_074](https://github.com/Ademola-coding/Blog_rails/assets/107632780/b824ec90-80a7-479f-b7b7-d68790bc8ca6)

- [ ] According to the project requirement, you are meant to fix the n+1 problem. The problem usually occurs when getting a users posts and subsequently fetching the comments. You are meant to update your `post index` method to fix that, then take a screenshot of the log when you visit /users/1/posts and add it to this pull request summary. I'll suggest you do that. :+1: :wave: